docs/validator/economics.md

```diff
@@ -0,0 +1,134 @@
+
+NEAR protocol determines validators at the end of each `epoch`, ordering them by the size of their stake. Current validators are re-elected automatically, by *re-staking* their tokens plus the accrued rewards; new validators may replace an existing one if they offer a more significant stake.
+
+As a validator, you can obtain a *seat* by:
+- Staking your NEAR tokens
+- Staking delegated NEAR tokens
+
+<blockquote class="info">
+    <strong>did you know?</strong><br><br>
+    A large enough stake can obtain more than one seat, multiplying the rewards. 
+</blockquote>
+
+### Stake your NEAR tokens
+
+Use [NEAR Shell](../development/near-clitool.md) to issue a staking transaction and append your transaction to the `proposals`. At the end of the `epoch`, if your proposal is sufficient, your node will become a validator - and will begin to generate chunks/blocks after one more epoch. You will be able to monitor the `proposals`, and eventually top-up your transaction to increase your odds.
+
+### Stake delegated NEAR tokens
+
+NEAR leverages *smart contracts* for delegation. Therefore, every validator should implement its staking contract, or use one already available among the community. 
+This option is still work in progress, so please follow the [NEP on GitHub](https://github.com/nearprotocol/NEPs/blob/staking-contract/text/0000-staking-contract.md) for the specifications, and add your comments in the 'Riskless Delegation' discussion on [NEAR's Commonwealth](https://commonwealth.im/near/proposal/discussion/357-riskless-delegation-aka-tezos-delegation).
+
+<blockquote class="warning">
+    <strong>heads up</strong><br><br>
+    In the beginning, the network will run on a limited number of shards, with all validators competing for a limited number of seats.
+    As a simple rule-of-thumb, distributing 40% of the total supply (400 million out of 1 billion NEAR tokens) between 100 seats will need an average of 4 million NEAR tokens per validator.
+</blockquote>
+
+
+## 2. Nodes infrastructure
+
+Once you obtained your NEAR stake, you need an infrastructure to serve the network securely.
+At the beginning of a new `epoch`, the protocol randomly assigns every validator to one of these roles:
+- Chunk Producer, for collecting transactions and applying them to the state
+- Block Producers, for generating the single blocks containing the current chunks 
+- Fishermen (aka 'hidden' validators), for challenging the state to verify the work of other validators
+
+Whether the case, every validator must provide **availability** and **consistency**.
+
+### Availability
+
+The protocol requires a `MINIMUM_THRESHOLD` of validating signatures, proportional to the median chunk/block producer in the `epoch` and the number of your *seats*. If your node falls below `90 %`, it will be *kicked out* at the end of the epoch and you will have to issue the staking `proposal` again.
+In practical terms, you have to:
+Wait for the end of your failed epoch to unlock your stake
```

Comment:
Doesn't have to wait until the end. As soon as you realize that you wouldn't be able to meet the threshold in the current epoch you can issue a new staking transaction.